doc/usecases/core/UC_keyset_delete.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Delete `Key` from `KeySet`
+
+## Summary
+
+- **Title:** Delete `Key` from `KeySet`
+- **Scope:** `libelektra-core`
+- **Level:** Developer Goal
+- **Actors:** Core, Caller
+- **Brief:** Caller removes `Key` from existing `KeySet`
```

Comment:
Also be consistent within the documents. If it is called remove, it must be called remove everywhere.

Reply:
Like I also said above, IMO a use case is a more informal, prose description. Unless explicitly stated or in a completely different context, you should be able to assume that normal English synonyms (like "remove" and "delete") have the same meaning.

In some sense, I see the use cases as a description of what Elektra can do. When I read them, I don't want to study a Glossary first to know what's going on.

However, as already stated above, in this case I'm open to renaming the use case to "Remove".